app/controllers/movies_controller.rb

```diff
@@ -1,5 +1,5 @@
 class MoviesController < ApplicationController
-  before_action :set_movie, only: %i[ show edit update destroy ]
```

Comment:
to be honest that was a good way to do it. It is recommend by rails guides. But we will get into that in the future. Right now it is very good. I will show you rubocop gem that automatically corrects our code :)